pwnlib/tubes/remote.py

```diff
@@ -71,8 +72,10 @@ def __init__(self, host, port,
             self.lhost, self.lport = self.sock.getsockname()[:2]
 
             if ssl:
-                self.sock = _ssl.wrap_socket(self.sock)
-
+                try:
+                    self.sock = _ssl.wrap_socket(self.sock, **ssl)
```

Comment:
You should check the type of `ssl` (e.g. instance of `dict`) rather than try-except.